OpInfo: Sample input cleanup (1/n)

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #84567
* #86324
* #86380
* #86379
* __->__ #86231

This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

peterbell10 committed:
OpInfo: Sample input cleanup (1/n)

This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Sample input cleanup (1/n)"


This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Sample input cleanup (1/n)"


This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Sample input cleanup (1/n)"


This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

[ghstack-poisoned]

peterbell10 committed:
Rebase on "OpInfo: Sample input cleanup (1/n)"


This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

[ghstack-poisoned]

mruberry reviewed (approved):
Stamped!

You'll still have to spot an opportunity to actually merge this, though!

peterbell10 committed:
Rebase on "OpInfo: Sample input cleanup (1/n)"


This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Sample input cleanup (1/n)"


This rewrites various sample and error input functions to:
- use the convention of `make_arg = functools.partial(make_tensor, ...)`
- use the new natural syntax for `SampleInput` construction
- yield instead of returning a lists, to reduce memory consumption

[ghstack-poisoned]
